### PR TITLE
feat (ui):  display job variable

### PR DESCRIPTION
**Pull request by sguiheux**

### CODE REVIEW

File: ui/src/app/views/run/workflow/variables/job.variables.component.ts [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+})
+export class JobVariableComponent {
+
+    @ViewChild('jobVariablesModal')
+    jobVariablesModal: SemanticModalComponent;
+
+    @Input('variables')
+    set variables( data: Array<Variable>) {
+        this.init();
+        if (data) {
```
Comment (bnjjj) [inline ui/src/app/views/run/workflow/variables/job.variables.component.ts]: `if (Array.isArray(data))` could be better

Comment (sguiheux) [inline ui/src/app/views/run/workflow/variables/job.variables.component.ts]: even if we use typescript?  just want to know if it's null or undefined, not if it's an array

Comment (sguiheux) [inline ui/src/app/views/run/workflow/variables/job.variables.component.ts]: data is already an array

File: ui/src/assets/i18n/en.json [hunk 2]
```diff
@@ -220,6 +221,16 @@
   "pipeline_deleted" : "Pipeline deleted",
   "pipeline_delete_label" : "Delete pipeline",
   "pipeline_delete_description" : "Once you delete a pipeline, there is no going back. Please be certain.",
+  "pipeline_job_variable_app" : "Application variables",
+  "pipeline_job_variable_build" : "Build variables",
+  "pipeline_job_variable_cds" : "CDS Variables",
+  "pipeline_job_variable_env" : "Environment variables",
+  "pipeline_job_variable_git" : "Git variables",
+  "pipeline_job_variable_parent" : "Parent pipleine variables",
```
Comment (yesnault) [inline ui/src/assets/i18n/en.json]: pipeline

File: ui/src/assets/i18n/fr.json [hunk 3]
```diff
@@ -220,6 +221,16 @@
   "pipeline_deleted" : "Pipeline supprimé",
   "pipeline_delete_label" : "Suppression du pipeline",
   "pipeline_delete_description" : "Une fois le pipeline supprimé, il n'y a pas de retour possible.",
+  "pipeline_job_variable_app" : "Variables d'application",
+  "pipeline_job_variable_build" : "Variables de build",
+  "pipeline_job_variable_cds" : "Variables CDS",
+  "pipeline_job_variable_env" : "Variables d'environnement",
+  "pipeline_job_variable_git" : "Variables git",
+  "pipeline_job_variable_parent" : "Variables  du pipeline parent",
```
Comment (yesnault) [inline ui/src/assets/i18n/fr.json]: too many space

**Pull request merged**